src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--31">
+      <div class="calendar__day calendar__day--date--1 calendar__day--start-day--sun"></div>
+      <div class="calendar__day calendar__day--date--2"></div>
+      <div class="calendar__day calendar__day--date--3"></div>
+      <div class="calendar__day calendar__day--date--4"></div>
+      <div class="calendar__day calendar__day--date--5"></div>
+      <div class="calendar__day calendar__day--date--6"></div>
+      <div class="calendar__day calendar__day--date--7"></div>
+      <div class="calendar__day calendar__day--date--8"></div>
+      <div class="calendar__day calendar__day--date--9"></div>
+      <div class="calendar__day calendar__day--date--10"></div>
+      <div class="calendar__day calendar__day--date--11"></div>
+      <div class="calendar__day calendar__day--date--12"></div>
```

Comment:
Do you really need modifiers for each calendar day?
I think you can just use loop and [nth-child preudoclass](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) to set the days using index of the loop